packages/react/src/callout/Callout.stories.tsx

```diff
@@ -185,7 +185,7 @@ export const NoTitle: Story = {
 	},
 };
 
-export const CustomTitleSize: Story = {
+export const customTitleElementSize: Story = {
```

Comment:
```suggestion
export const CustomTitleElementSize: Story = {
```

Reply:
lol find and replace. I shouldn't have changed this file 🤦‍♂️